webapi/_test_data/all_data/anything-data.ttl

```diff
@@ -20,7 +20,7 @@
 	knora-base:isDeleted false ;
 	knora-base:attachedToUser <http://data.knora.org/users/9XBCrDV3SRa7kS1WwynB4Q> ;
 	knora-base:attachedToProject <http://data.knora.org/projects/anything> ;
-	rdfs:label "Ein Ding für wen, der die Dinge liebe" ;
+	rdfs:label "Ein Ding für wen, dem die Dinge gefallen" ;
```

Comment:
für **jemanden**, dem

the new generation of triplestores will do a grammar check and randomly reject transactions if the found a violation of German grammar